src/style.css

```diff
@@ -0,0 +1,144 @@
+* {
+  margin: 0;
+}
+
+:root {
+  --grey: #616070;
+  --blue: #00acdc;
+  --darkblue: #060b35;
+  --white: #fff;
+}
+
+html {
+  font-family: "Roboto", sans-serif ;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: var(--white);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px ;
+  height: 134px;
+  padding-left: 19px;
+  padding-top: 32px;
+}
+
+.card__text {
+  padding: 0 16px;
+}
+
+.card__descriprion {
+  font-size: 12px;
+  color: var(--darkblue);
+  margin-top: 36px;
+  line-height: 18px;
+  width: 166px;
+  font-weight: 500;
+}
+
+.card__productCode {
+  margin-top: 4px;
+  font-size: 10px;
+  font-weight: 400;
+  color: var(--grey);
+  line-height: 14px;
+}
+
+.card__firstRow {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 16px;
+}
+
+.card__reviews {
+  color: var(--darkblue);
+  font-weight: 400;
+  font-size: 10px;
+  text-align: end;
+  margin-top: 2px;
+}
+
+.card__secondRow {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  margin-bottom: 16px;
+  align-items: center;
+  line-height: 18px;
+}
+
+.card__price {
+  font-size: 12px;
+  color: var(--grey);
+  align-self: flex-start;
+}
+
+.card__amount {
+  color: var(--darkblue);
+  font-weight: 700;
+  font-size: 16px;
+  align-self: flex-end  ;
+
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url(images/star.svg) center no-repeat;
+  height: 16px;
```

Comment:
You can remove the 'margin-right: 4px;' on this line as it's already defined in the .stars__star:not(:last-child) rule on line 101. This way, you avoid redundancy and keep the code clean 🧹